### PR TITLE
Added pounds and kilograms to !convert

### PR DESCRIPTION
**Pull request by inkdracula**
I would love to add the ability to convert pounds and kg using !convert, so i wrote the extra code for it.

### CODE REVIEW

File: sintia/sintia.py [hunk 1]
```diff
@@ -503,6 +503,8 @@ def format_quote(self, quote: Quote) -> str:
             'k': 'kelvin',
             'mi': 'miles',
             'km': 'kilometers',
+            'lbs': 'pounds',
+            'kg': 'kilograms',
```
Comment (LiraNuna) [inline sintia/sintia.py:507]: wanna add just kilos as well? You don't have to

Comment (inkdracula) [reply sintia/sintia.py:507]: no, i think everyone can understand kg or kilograms instead of kilos

Comment (LiraNuna) [reply sintia/sintia.py:507]: Why not both?

Comment (inkdracula) [reply sintia/sintia.py:507]: :|

**Pull request merged**